Project-1/Praoject_1_final_version

```diff
@@ -0,0 +1,384 @@
+	For statements coming after ∃x∀y∃z∀w, there must be an x and a w included in them while there is no necessity of them having a y or w. Also, for such statement to be true, they must be true in all conditions of y and all conditions of w (T or F), but as long as theyr are true for at least one case of x and one case of z they are fine.
+```
+	
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+
+{(a,b),(b,a),(a,a)}
+picture was not uploaded
+
+```
+	
+8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+  ```
+  (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  
+	1.	¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+	2.	¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))	(1)
+	3.	¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))	(2)
+	4.		¬(Pa ∧ Qa)		(3)
+	5.		¬(¬Pa ∨ ¬Qa)		(3)
+	6. 		¬¬Pa			(5)
+	7.		¬¬Qa			(5)
+	8.		Pa			(6)
+	9.		Qa			(7)
+			      /\
+		       /  \
+		      /    \
+		     /	    \
+		    /        \
+		   /          \
+		  /            \
+	10.    ¬Pa (4)		11. ¬Qa (4)
+  
+  (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  
+  	1.	¬∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+	2.	¬∀y(¬(Pa ∨ Qa) → (¬Pa ∧ ¬Qa))	(1)
+	3.	¬(¬(Pa ∨ Qa) → (¬Pa ∧ ¬Qa))	(2)
+	4.		¬(Pa ∨ Qa)		(3)
+	5.		¬(¬Pa ∧ ¬Qa)		(3)
+	6. 		¬Pa			(4)
+	7.		¬Qa			(4)
+			      /\
+		       /  \
+		      /    \
+		     /	    \
+		    /        \
+		   /          \
+		  /            \
+		 /	          	\
+	8. 	¬¬Pa (5)	9. ¬¬Qa (5)
+	10.	Pa  (8)		11. Qa	(9)
+		X		    X
+  
+  (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  	
+	1.	¬∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+	2.	¬∀y((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))	(1)
+	3.	¬((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))	(2)
+	4.		¬Pa ∨ ¬Qa		(3)
+	5.		¬¬(Pa ∧ Qa)		(3)
+	6.		Pa ∧ Qa			(5)
+	7.		Pa			(6)
+	8.		Qa			(6)
+			      /\
+		       /  \
+		      /    \
+		     /	    \
+		    /        \
+		   /          \
+		  /            \
+		 /		          \
+	9.	¬Pa  (4) 	10. ¬Qa  (4)
+		X		    X
+	
+  (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  	
+	1.	¬∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+	2.	¬∀y((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))	(1)
+	3.	¬((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))	(2)
+	4.	¬Pa ∧ ¬Qa(3)5.¬¬(Pa ∨ Qa)	(3)
+	6.		Pa ∨ Qa			(5)
+	7.		¬Pa			(4)
+	8.		¬Qa			(4)
+		      	/\
+		       /  \
+		      /    \
+		     /	    \
+		    /        \
+		   /          \
+		  /            \
+		 /		          \
+	9.	Pa  (6)		10. 	Qa  (6)
+		X			X
+```
+	
+9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
+	
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
```

Comment:
You need to instantiate the first universal first, then discharge it to prove what I've asked. 
You also should remove the ∀y since it is irrelevant.